File: src/test/java/day2/CubeConundrumTest.java
package day2;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class CubeConundrumTest {

    @Test
    void firstStar() {
        CubeConundrum cubeConundrum = new CubeConundrum();
        assertEquals(8, cubeConundrum.firstStar());
    }

    @Test
    void secondStar() {
        CubeConundrum cubeConundrum = new CubeConundrum();
        assertEquals(2286, cubeConundrum.secondStar());
    }
}

File: src/test/kotlin/day02/CubeConundrumTest.kt
package day02

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

internal class CubeConundrumTest {
    @Test
    fun firstStar() {
        assertEquals(8, CubeConundrum().firstStar())
    }

    @Test
    fun secondStar() {
        assertEquals(2286, CubeConundrum().secondStar())
    }
}